packages/cli/src/commands/new/project.ts

```diff
@@ -126,54 +132,54 @@ export const parseConfig = async (
   flags: Partial<ProjectInitializerConfig>,
   boosterVersion: string
 ): Promise<ProjectInitializerConfig> => {
-  if (flags.default) {
+  if (flags.interactive) {
+    const description = await prompter.defaultOrPrompt(
+      flags.description,
+      'What\'s your project description? (default: "")'
+    )
+    const versionPrompt = await prompter.defaultOrPrompt(flags.version, "What's the first version? (default: 0.1.0)")
+    const version = versionPrompt || '0.1.0'
+    const author = await prompter.defaultOrPrompt(flags.author, 'Who\'s the author? (default: "")')
+    const homepage = await prompter.defaultOrPrompt(flags.homepage, 'What\'s the website? (default: "")')
+    const licensePrompt = await prompter.defaultOrPrompt(
+      flags.license,
+      'What license will you be publishing this under? (default: MIT)'
+    )
+    const license = licensePrompt || 'MIT'
+    const repository = await prompter.defaultOrPrompt(
+      flags.repository,
+      'What\'s the URL of the repository? (default: "")'
+    )
+    const providerPackageName = await getProviderPackageName(prompter, flags.providerPackageName)
+
+    return Promise.resolve({
+      projectName: flags.projectName as string,
+      providerPackageName,
+      description,
+      version,
+      author,
+      homepage,
+      license,
+      repository,
+      boosterVersion,
+      skipInstall: flags.skipInstall || false,
+      skipGit: flags.skipGit || false,
+      interactive: true,
+    })
+  } else {
```

Comment:
Given that the function is async we can actually just return the object as is. Could you change the return in the `else` too please? 🙏